app/MMB/src/context/BookingDetailContext.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import * as React from 'react';
+
+const defaultState = {
+  isPastBooking: false,
+  actions: {
+    setIsPastBooking: () => {},
+  },
+};
+
+const { Consumer, Provider: ContextProvider } = React.createContext({
+  ...defaultState,
+});
+
+type Props = {|
+  children: React.Node,
```

Comment:
Can we start to use read-only props and states more? https://flow.org/en/docs/types/interfaces/#interface-property-variance-read-only-and-write-only-

Reply:
Yes, I guess that makes sense 🙂

Reply:
Added readonly types

Reply:
Please try to do it everywhere in the future.